controllers/CollectionController.js

```diff
@@ -0,0 +1,122 @@
+const Bluebird = require("bluebird")
+
+const {
+  ConflictError,
+  protectedFolderConflictErrorMsg,
+} = require("@errors/ConflictError")
+
+const GitHubService = require("@services/db/GitHubService")
+const CollectionPageService = require("@services/fileServices/MdPageServices/CollectionPageService")
+const ThirdNavPageService = require("@services/fileServices/MdPageServices/ThirdNavPageService")
```

Comment:
Could we rename `ThirdNavPageService` to `SubCollectionPageService` for clarity? What do y'all think?